apps/src/music/ai/constants.ts

```diff
@@ -0,0 +1,139 @@
+// Midi drum notes that correspond to the 8-supported drum types in Music Lab pattern blocks.
+export const musiclabMidiDrumNotes = {
```

Comment:
Great refactor and reorganization! This is much easier to parse and understand. 

Just dropping a comment so I don't forget - post-HOC, it would be ideal to just include these notes directly in our Library JSON, rather than using index; currently, we treat drum samples as note-based samples, because that's how we setup the sampler map for the ToneJS sample, but the note value is just the index of the sample in the list of sounds (i.e. that's why kick drum is 0, snare is 1, etc). If we add an explicit note field and use these values, then we have even less ambiguity and fewer assumptions (don't need to know that our drums are restricted to the range 0-7)